### PR TITLE
Feature/9 completed section destination

### PR DESCRIPTION
**Pull request by thuan734655**
completed section destination

### CODE REVIEW

File: src/Pages/index.html
Comment (DatWithTheWorld) [inline src/Pages/index.html:1]: end of lines

File: src/components/box-section-desetination.html
Comment (DatWithTheWorld) [inline src/components/box-section-desetination.html:1]: same

File: src/stylesheet/Layout/_destination.scss
Comment (DatWithTheWorld) [inline src/stylesheet/Layout/_destination.scss:1]: define parameter for color

Comment (DatWithTheWorld) [reply src/stylesheet/Layout/_destination.scss:1]: don't loop your code in mixin responsive. You should use mixin responsive where your values are changed.

Comment (thuan734655) [reply src/stylesheet/Layout/_destination.scss:1]: I fixed it. Thanks

File: src/Layout/main.html
Comment (DatWithTheWorld) [inline src/Layout/main.html:1]: wrap your img tags with figure tag

Comment (thuan734655) [reply src/Layout/main.html:1]: I fixed it

File: src/Pages/index.html
Comment (thuan734655) [reply src/Pages/index.html:1]: I fixed it

File: src/components/box-section-desetination.html
Comment (thuan734655) [reply src/components/box-section-desetination.html:1]: i fixed it

**Pull request closed without merging**